clar2wasm/src/standard/standard.wat

```diff
@@ -1409,29 +1409,81 @@
+        ;;          2. Leave last 8 bytes to store the length 
+        ;;          3. Fill the remaining space with zeros
+        ;;          4. Consider following 
+        ;;              a. if length=13 bytes, 1 byte to add after data, last 8 bytes for length, 64-(13+1+8)= 42 bytes of zeros
+        ;;              b. if length=123 bytes, 1 byte to add after data, last 8 bytes for length, 123+1+8 = 132, which is obviously 
+        ;;                 greater than 128, so next number which is divisible by 64 is 192. So 192-(123+1+8) = 60 bytes of zeros
+        ;;                 between 1 added after the data and last 8 bytes of length
+        ;;          5. Calculate length in bits (i.e. 13x8=104)
+        ;; 4. Before calculating hash and after padding sha256 data, algorithm needs to calculate total 64 words. 16 words are
+        ;;    obtained directly from the first 64 bytes of padded data. As each word is of 4 bytes, 16 words will require 64 bytes.
+        ;; 5. Next word will be calculated as w16 = w0 + σ0 + w9 + σ1 (w15 would be the 16th word as we are starting from 0). 
+        ;;    To take a look at how these are working, please visit https://sha256algorithm.com/
+        ;; 6. After calculating w63 (64th word), data is ready to perform hashing rounds.
+        ;; 7. Hashing rounds involve 8 variables, a,b,c,d,e,f,g and h. Each variable is of 4 bytes, hence sha256 will produce 32 bytes
+        ;;    result.
+        ;; 8. To take a look at how these words are being calculated, take a look at https://sha256algorithm.com
+        ;; 9. Finally result is being stored at specific location.
+              
+        ;; Index to track sha256 blocks
         (local $i i32)
-        ;; see this for an explanation: https://sha256algorithm.com/
 
+        ;; This function is responsible to add padding and length to the data, it returns total length divisible by 64
         (call $extend-data (local.get $offset) (local.get $length))
         (local.set $length)
-
+        
+        ;; Index to track all blocks
         (local.set $i (i32.const 0))
+        ;; Iterations on SHA-256 data (in 64 byte chunks/blocks)
         (loop
+            ;; Calculates next 48 words
             (call $block64 (local.get $i))
+
+            ;; Calculate hash using initial values and k-constants (64 rounds total)
             (call $working-vars)
             (br_if 0
                 (i32.lt_u
+                    ;; As each block is of 64 bytes, we'll increment by 64
                     (local.tee $i (i32.add (local.get $i) (i32.const 64)))
                     (local.get $length)
                 )
             )
         )
 
-        ;; store at result position with correct endianness
+        ;; Reversing each word (4 bytes) because v128.store uses little-endian format but we want it in big-endian format
+        ;; v128.store can only store 16 bytes, so for 32 bytes, two operations with proper offsets are needed
```

Comment:
That's a lot of words for "handling endianness".

Reply:
I was stuck at this point because I was a new comer. Remember I asked you a question about this? Are we assuming that next one will be expert in this?

Reply:
I would say that it was a weird questions to begin with. Normally, people should know that Webassembly store in and load from memory as little-endian.

Reply:
Alright. I was new to WASM and it just slipped out of my mind. Is it safe to remove these comments? because we're assuming that person has a good experience.